packages/ui/src/services/api/canUseEndpoint.ts

```diff
@@ -1,12 +1,12 @@
-import { USE_API, USE_API_CASE_ENDPOINT, USE_API_CASES_INDEX_ENDPOINT } from "config"
+import { FORCES_WITH_API_ENABLED, USE_API, USE_API_CASE_ENDPOINT, USE_API_CASES_INDEX_ENDPOINT } from "config"
 
 export enum ApiEndpoints {
   CaseDetails = "CaseDetails",
   CaseList = "CaseList"
 }
 
-export const canUseApiEndpoint = (endpoint: ApiEndpoints): boolean => {
-  if (!USE_API) {
+export const canUseApiEndpoint = (endpoint: ApiEndpoints, forces: string[]): boolean => {
+  if (!USE_API || !forces.some((force) => FORCES_WITH_API_ENABLED.has(force))) {
```

Comment:
Can we split these checks out? 

We want `USE_API` to be a global enable for everyone, regardless of whether the force can use the API.